Source/Flow/Private/FlowAsset.cpp

```diff
@@ -341,6 +425,56 @@ UFlowNode_CustomInput* UFlowAsset::TryFindCustomInputNodeByEventName(const FName
 	return nullptr;
 }
 
+UFlowNode_CustomOutput* UFlowAsset::TryFindCustomOutputNodeByEventName(const FName& EventName) const
+{
+	for (const TPair<FGuid, UFlowNode*>& Node : Nodes)
+	{
+		if (UFlowNode_CustomOutput* CustomOutput = Cast<UFlowNode_CustomOutput>(Node.Value))
+		{
+			if (CustomOutput->GetEventName() == EventName)
+			{
+				return CustomOutput;
+			}
+		}
+	}
+
+	return nullptr;
+}
+
+TArray<FName> UFlowAsset::GatherCustomInputNodeEventNames() const
```

Comment:
The "name" versions of the custom input/output are editor-only, so if the names of inputs/outputs are used at runtime, we need to gather them from the nodes list.  These functions handle that work.